controller/config.go

```diff
@@ -73,6 +72,13 @@ const (
 	// detault
 	MongoMemoryProfile = "mongo-memory-profile"
 
+	// LogPruneIntervalDays is the pruning interval for log entries.
+	LogPruneIntervalDays = "logs-prune-interval-days"
```

Comment:
Done, but sadly ParseDuration(0 only supports hours as the maximum unit. Too bad it doesn't support "days".